internal/service/rds/instance.go

```diff
@@ -2821,6 +2822,41 @@ func waitDBInstanceAvailable(ctx context.Context, conn *rds.Client, id string, t
 	return nil, err
 }
 
+func waitDBInstanceStopped(ctx context.Context, conn *rds.Client, id string, timeout time.Duration) (*types.DBInstance, error) {
+	stateConf := &retry.StateChangeConf{
+		Pending: []string{
+			instanceStatusBackingUp,
+			instanceStatusConfiguringEnhancedMonitoring,
+			instanceStatusConfiguringIAMDatabaseAuth,
+			instanceStatusConfiguringLogExports,
+			instanceStatusCreating,
+			instanceStatusMaintenance,
+			instanceStatusModifying,
+			instanceStatusMovingToVPC,
+			instanceStatusRebooting,
+			instanceStatusRenaming,
+			instanceStatusResettingMasterCredentials,
+			instanceStatusStarting,
+			instanceStatusStopping,
+			instanceStatusStorageFull,
+			instanceStatusUpgrading,
+		},
+		Target:     []string{instanceStatusStopped},
+		Refresh:    statusDBInstance(ctx, conn, id),
+		Timeout:    timeout,
+		Delay:      10 * time.Second,
+		MinTimeout: 3 * time.Second,
```

Comment:
```suggestion
		Target:                    []string{instanceStatusStopped},
		Refresh:                   statusDBInstance(ctx, conn, id),
		Timeout:                   timeout,
		ContinuousTargetOccurence: 2,
		Delay:                     10 * time.Second,
		MinTimeout:                3 * time.Second,
```

Might consider ContinousTargetOc... Sometimes the only thing that avoids eventual consistency problems.